Source/ThirdParty/GltfPipeline/addDefaults.js

```diff
@@ -214,6 +214,34 @@ define([
         }
     }
 
+    function getAnimatedNodes(gltf) {
+        var nodes = {};
+        ForEach.animation(gltf, function(animation) {
+            ForEach.animationChannel(animation, function(channel) {
+                var target = channel.target;
+                var node = target.node;
+                var path = target.path;
+                // Ignore animations that target 'weights'
+                if (path === 'translation' || path === 'rotation' || path === 'scale') {
+                    nodes[node] = true;
```

Comment:
Just for readability's sake, can we rename the index variable here?  Such as `nodes[nodeId] = true;` or similar.